cloudify_agent/api/utils.py

```diff
@@ -160,7 +167,7 @@ def get_home_dir(username=None):
         if username is None:
             return os.path.expanduser('~')
         else:
-            return os.path.expanduser('~{0}'.format(username))
+            return get_windows_home_dir(username)
```

Comment:
It was working properly only when home directories of currently logged user and user we are asking for were in the same directory.
It is not the case (by default) when one of those users is standard local user and the other one is the user that runs services (some local admin account I believe) - which happened to be the case when trying to install new agents on windows via an old agent.